bin/mail/2017-01-12-csp-csd-interested-teachers/test-params.csv

```diff
@@ -0,0 +1,3 @@
+email,name
+andrew@code.org,Andrew Oberhardt
+sarah.fairweather@code.org,Sarah Fairweather
```

Comment:
Perhaps also `asher@code.org, Asher Kach` if you want my eyes on the email. Probably you don't. :)